src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { FcManager } from 'react-icons/fc';
+import styled from 'styled-components';
+import InputSearch from '../InputSearch';
+
+const NavBarStyled = styled.div`
+  background-color: ${(props) => props.theme.primary};
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+`;
+
+function NavBar({ search }) {
```

Comment:
I also recommend using the context hook to obtain the search value instead of using prop drilling to obtain the value.